debian/rules

```diff
@@ -176,9 +176,6 @@ endif
 	# Move test plugins that are only needed by the client to the libmariadb path
 	mv -v $(TMP)/usr/lib/mysql/plugin/qa_auth_client.so $(TMP)/usr/lib/$(DEB_HOST_MULTIARCH)/libmariadb3/plugin/
 
-override_dh_installlogrotate-arch:
-	dh_installlogrotate --name mysql-server
```

Comment:
Should this be mariadb.logrotate as it's renamed?

Reply:
These lines are deleted. Can't change the a string and delete it at the same time..